README.md

```diff
@@ -94,97 +95,86 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
+> Key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Add new categories and delete it**
+- **Add new transactions and delete it**
+- **Display details of transactions**
+- **You can see how much money you spent**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- [Live Demo Link](https://hana-kumie-1212.herokuapp.com/)
```

Comment:
- [ ] Kindly fix your deployment. I see nothing when I access the link.

  ![fffy](https://user-images.githubusercontent.com/54848363/209113354-edfa16cc-037c-4bf9-80b6-27c0213e0639.jpg)

Reply:
I have deployed it. But Heroku becomes paid. Because of this, it does not work.

Reply:
I have deployed but Heroku stops free deployment services. I have no paid deployment account...

Reply:
Hi @Kumilachew-g - [ ] Kindly see this article on [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/10903290796179) how how to deploy your application on render